src/Behat/Gherkin/Parser.php

```diff
@@ -83,7 +85,7 @@ public function parse($input, $file = null)
         while ('EOS' !== ($predicted = $this->predictTokenType())) {
             $node = $this->parseExpression();
 
-            if ($node === null || $node === "\n") {
```

Comment:
I checked and couldn't find evidence of any parser method returning a null (in addition, the PHPDocs never mentioned nulls either, hence the PHPStan warning).